From 4a22852e86620d708d8c5b2ce342291f955c8fa4 Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 4 May 2025 00:51:18 +0200
Subject: [PATCH 20/20] Allow keeping Android app in background with
 QtQuickView

Do not terminate Qt as this would stop the main Qt thread which is not
desirable when the app is supposed to keep running in the background (and
still supposed to process events via the Qt event loop).

Change-Id: I18e2c2a054b19bdbdbe03d189f9a36dcdbe1e204
---
 .../jar/src/org/qtproject/qt/android/QtEmbeddedDelegate.java     | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/android/jar/src/org/qtproject/qt/android/QtEmbeddedDelegate.java b/src/android/jar/src/org/qtproject/qt/android/QtEmbeddedDelegate.java
index fbfe71dd1a8..31cc04fa5fe 100644
--- a/src/android/jar/src/org/qtproject/qt/android/QtEmbeddedDelegate.java
+++ b/src/android/jar/src/org/qtproject/qt/android/QtEmbeddedDelegate.java
@@ -75,7 +75,6 @@ class QtEmbeddedDelegate extends QtActivityDelegateBase
                     m_activity.getApplication().unregisterActivityLifecycleCallbacks(this);
                     QtNative.unregisterAppStateListener(QtEmbeddedDelegate.this);
                     QtEmbeddedViewInterfaceFactory.remove(m_activity);
-                    QtNative.quitQt();
                     QtNative.setActivity(null);
                 }
             }
-- 
2.49.0

